marketplace-service/src/test/java/com/axonivy/market/service/impl/MetadataServiceImplTest.java

```diff
@@ -45,6 +51,32 @@ class MetadataServiceImplTest extends BaseSetup {
   @Mock
   private MetadataSyncRepository metadataSyncRepo;
 
+  @Test
+  void testUpdateArtifactAndMetaDataForProduct() {
+    ProductJsonContent mockProductJsonContent = getMockProductJsonContent();
+    mockProductJsonContent.setId("amazon-comprehend-9.2.0");
+    mockProductJsonContent.setProductId("amazon-comprehend");
+
+    Artifact mockArtifact = Artifact.builder()
```

Comment:
- we do have mockArtifact in BaseSetup, you can reuse it
- same with mockProductJsonContent (MOCK_PRODUCT_ID_WITH_VERSION and MOCK_ARTIFACT_ID)

Reply:
I saw you still use hard-code
![image](https://github.com/user-attachments/assets/caf9b921-11f0-48b1-953b-bfb662337eab)